test/test_helper.rb

```diff
@@ -2,3 +2,21 @@
 
 require "yast"
 require "yast/rspec"
+
+def stub_root_part
```

Comment:
np: Maybe you would like to follow [this approach](https://github.com/yast/yast-installation/blob/1c225041d00b0db4af3497236c99b7a9beeb973d/test/test_helper.rb#L64) to not pollute the `test_helper` file.